libandroid-navigation/src/test/java/com/mapbox/services/android/navigation/v5/navigation/DownloadUpdateListenerTest.java

```diff
@@ -0,0 +1,45 @@
+package com.mapbox.services.android.navigation.v5.navigation;
+
+import org.junit.Test;
+
+import java.io.File;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+
+public class DownloadUpdateListenerTest {
+
+  @Test
+  public void onFinishedDownloading_tarIsUnpacked() {
+    RouteTileDownloader downloader = mock(RouteTileDownloader.class);
```

Comment:
Could you extract (if possible) some of the shared arrange code across these tests into a `private` factory method?